samples/graphene-hello/preload/README.md

```diff
@@ -1,49 +1,42 @@
+# Graphene "Hello World!" example (preload)
+This example shows how to run a [Graphene](https://github.com/oscarlab/graphene) application in Marblerun. In essence, you have to add the `premain` process as a preloaded library for LD_PRELOAD. `premain` will contact the Coordinator, set up the environment, and run the actual application. See the commented [hello.manifest.template](hello.manifest.template) for details.
 ## Requirements
-First, get Graphene up and running. You can use either the [Building](https://graphene.readthedocs.io/en/latest/building.html) or [Cloud Deployment](https://graphene.readthedocs.io/en/latest/cloud-deployment.html) guide to build and initially setup Graphene.
+First, get Graphene up and running. You can use either the [Building](https://graphene.readthedocs.io/en/latest/building.html) or [Cloud Deployment](https://graphene.readthedocs.io/en/latest/cloud-deployment.html) guide to build and initially setup Graphene. You will need hardware with Intel SGX support.
 
-Then, before you can run the sample, make sure you got the prerequisites for ECDSA remote attestation installed on your system. You can collectively install them with the following command:
+Then, before you can run the example, make sure you got the prerequisites for ECDSA remote attestation installed on your system. You can collectively install them with the following command:
 ```sh
 sudo apt install libsgx-quote-ex-dev
 ```
 
 ## Build
-You can build the sample as follows:
+You can build the example as follows:
 ```sh
 export GRAPHENEDIR=[PATH To Your Graphene Folder]
 make
 ```
 Then get `mr_enclave` from the build output and set it as `UniqueID` in `manifest.json`.
 
-Note that compared to the `spawn` method for this sample, the argv arguments are defined during the build and cannot be changed afterwards without rebuilding your Graphene application. To adjust the provisioned argv arguments, you can change the entry `hello.argv` in the Makefile.
+Note that compared to the `spawn` method for this example, the argv arguments are defined during the build and cannot be changed afterwards without rebuilding your Graphene application. To adjust the provisioned argv arguments, you can change the entry `hello.argv` in the Makefile.
 
 ## Run
-We assume that the Coordinator is run with the following environment variables:
-
-- EDG_COORDINATOR_MESH_ADDR=localhost:2001
-- EDG_COORDINATOR_CLIENT_ADDR=localhost:4433
-- EDG_COORDINATOR_DNS_NAMES=localhost
-- EDG_COORDINATOR_SEAL_DIR=$PWD
+Next, use the `erthost` command to start the Coordinator in a local enclave:
+```sh
+erthost ../../../build/coordinator-enclave.signed
+```
 
-Once the [Coordinator instance is running](../../BUILD.md#run-the-coordinator), upload the manifest to the Coordinator:
+The Coordinator exposes two APIs, a client API to instruct the Coordinator (port 4433) and a mesh API to communicate with your Marble (port 2001).
```

Comment:
I think the wording "The coordinator exposes... to instruct the coordinator" is a bit confusing. 

Maybe we can reword this a bit, highlight it's a REST API which an admin can use to manage the coordinator (e.g. setting a manifest) or something like that. Opinions?